rdrf/rdrf/services/rest/views/api_views.py

```diff
@@ -381,6 +384,8 @@ def get(self, request, task_id):
             res = AsyncResult(task_id)
             if res.ready():
                 cae = CustomActionExecution.objects.get(task_id=task_id)
+                if cae.user.username != request.user.username and not request.user.is_superuser:
```

Comment:
above the raise , add logger.info(f"illegal task access: user {request.user.username} task {task_id}")
same for the raise below for download